src/leaflike/bookmarks/validator.clj

```diff
@@ -1,31 +1,31 @@
-(ns leaflike.validator
+(ns leaflike.bookmarks.validator
   (:import org.apache.commons.validator.UrlValidator))
 
-(defn is-id?
+(defn id?
   [body]
   (let [id (:id body)]
     (and (some? id)
          (number? (bigdec id)))))
 
-(defn is-title?
+(defn title?
   [body]
   (contains? body :title))
 
-(defn is-url?
+(defn url?
   [body]
   (let [validator (UrlValidator.)
         url (:url body)]
     (and (some? url)
          (.isValid validator url))))
 
-(defn is-valid-bookmark?
+(defn valid-bookmark?
   [body]
-  (and (is-id? body)
-       (is-title? body)
-       (is-url? body)))
+  (and ((complement id?) body)
```

Comment:
Why the `(complement id?)` fn?
It can return true for invalid ids as well as no id being present. The latter seems to be your intention, right?